src/common/Serialize.h

```diff
@@ -31,7 +31,18 @@ SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #ifndef COMMON_SERIALIZATION_H_
 #define COMMON_SERIALIZATION_H_
 
+#include <limits>
```

Comment:
These should already be included in common/Common.h no?

Reply:
Well, we don't seem to use the precompiled header in cgame or sgame, so I can't assume everything is included.